packages/kit/types/ambient.d.ts

```diff
@@ -174,6 +174,10 @@ declare module '$app/navigation' {
 	/**
 	 * Returns a Promise that resolves when SvelteKit navigates (or fails to navigate, in which case the promise rejects) to the specified `url`.
 	 *
+	 * Note that navigating to another page that reuses the current `+page.svelte` will not remount the component.
+	 * Also note that not all `load` functions might be run again.
```

Comment:
Seems like slightly awkward phrasing - maybe this reads better?
```suggestion
	 * Also note that some `load` functions will not be re-run after a navigation.
```

Reply:
"will" sounds like it would be certain, which it isn't. I want that "maybe, maybe not"-notion be present in the sentence, but I agree it could be phrased better.

Reply:
Maybe this then?

```suggestion
	 * Also note that some `load` functions might not be re-run after a navigation.
```